app/view_models/admin/user_auction_view_model.rb

```diff
@@ -0,0 +1,77 @@
+class Admin::UserAuctionViewModel
+  attr_reader :auction, :user
+
+  DATE_FORMAT = '%m/%d/%Y'.freeze
+  NA_RESPONSE_STRING = '-'
+
+  def initialize(auction, user)
+    @auction = auction
+    @user = user
+  end
+
+  def title
+    auction.title
+  end
+
+  def id
+    auction.id
+  end
+
+  def start_date
+    DcTimePresenter.convert_and_format(auction.started_at, DATE_FORMAT, timezone_label: false)
```

Comment:
I am doing this so I will show it in ET (even though the started_at is stored in UTC)